test/unit/api/api-get-linking-metadata.test.js

```diff
@@ -59,8 +48,6 @@ tap.test('Agent API - getLinkingMetadata', (t) => {
   })
 
   t.test('should return available fields, no DT data, when DT enabled - no transaction', (t) => {
-    agent.config.distributed_tracing.enabled = true
```

Comment:
Minor: Given test is for DT enabled specifically, could have left explicit set in here which would avoid issues if the default ever changed again and makes test setup obvious.